torch/csrc/autograd/variable.cpp

```diff
@@ -125,8 +133,13 @@ DifferentiableViewMeta::~DifferentiableViewMeta() {
   base_.reset();
 }
 
+namespace {
+  std::shared_ptr<Node> singleton_shared_ptr;
+}
+
 const std::shared_ptr<Node>& Variable::grad_fn() const {
   if (is_view()) {
+    // NB: is_view() ==> get_autograd_meta()
```

Comment:
Does this means that we are sure that if `is_view() == true` then `get_autograd_meta != nullptr`?

Reply:
Yep that's right